### DISCUSSION TITLE
TOTP MFA support

### DISCUSSION THREAD

**Discussion by ghost** [firebase/flutterfire#11114] [Feature request]
To support TOTP MFA as documented at https://cloud.google.com/identity-platform/docs/admin/enabling-totp-mfa

**Comment by cbenhagen** [comment]
This should be fixed in firebase_auth - v4.9.0:
FEAT(auth): TOTP (time-based one-time password) support for multi-factor authentication (https://github.com/firebase/flutterfire/issues/11420). ([3cc1243c](https://github.com/firebase/flutterfire/commit/3cc1243c94368de44d3a5c4be96b905a0a37b963))

**Comment by ghost** [comment]
Any updates to this please? Can see https://pub.dev/packages/firebase_auth/changelog#490 but there are no docs? Does this support Windows, macOS, iOS and Android?
